--- /usr/lib/python3/dist-packages/UpdateManager/UpdateManager.py.orig	2025-06-25 15:15:56.941726136 +0300
+++ /usr/lib/python3/dist-packages/UpdateManager/UpdateManager.py	2025-06-25 15:41:15.808882645 +0300
@@ -27,7 +27,7 @@
 from gi.repository import Gio
 from gi.repository import GLib
 from gi.repository import GObject
-import uaclient.api.u.pro.packages.updates.v1 as ua
+# import uaclient.api.u.pro.packages.updates.v1 as ua
 
 GdkX11  # pyflakes
 
@@ -281,6 +281,7 @@
     def _get_ua_security_status(self):
         self.ua_security_packages = []
         self.ua_updates = []
+        return
         t = threading.Thread(target=self._fetch_ua_updates, daemon=True)
         t.start()
         while t.is_alive():
