--- /usr/lib/python3/dist-packages/UpdateManager/UpdateManager.py.orig	2025-06-25 15:15:56.941726136 +0300
+++ /usr/lib/python3/dist-packages/UpdateManager/UpdateManager.py	2025-06-25 15:43:17.614605707 +0300
@@ -27,7 +27,7 @@
 from gi.repository import Gio
 from gi.repository import GLib
 from gi.repository import GObject
-import uaclient.api.u.pro.packages.updates.v1 as ua
+# import uaclient.api.u.pro.packages.updates.v1 as ua
 
 GdkX11  # pyflakes
 
@@ -463,7 +463,7 @@
 
         self._check_oem_metapackages()
 
-        self._get_ua_security_status()
+        # self._get_ua_security_status()
 
         for pkgname in self.oem_metapackages:
             try:
